packages/workspace/src/schematics/init/init.ts

```diff
@@ -329,28 +329,64 @@ function setUpCompilerOptions(
   return tsconfig;
 }
 
+function renameSync(tree: Tree, from: string, to: string, cb: (err: string) => void) {
```

Comment:
These are amazing!

Can you please move this to `/packages/workspace/src/utils/ast-utils.ts`

Reply:
Thanks, done!